test/10-verify.spec.js

```diff
@@ -21,20 +20,33 @@ import {invalidContexts} from './contexts/index.js';
 import {credential as mockCredential} from './mocks/credential.js';
 import {assertionController} from './mocks/assertionController.js';
 import {VeresOneDriver} from 'did-veres-one';
+import {
+  CONTEXT_V1 as odrlCtx,
+  CONTEXT_URL_V1 as odrlCtxUrl
+} from '@digitalbazaar/odrl-context';
+import {
+  CONTEXT_V1 as vcExamplesV1Ctx,
+  CONTEXT_URL_V1 as vcExamplesV1CtxUrl
+} from '@digitalbazaar/credentials-examples-context';
+
+const contexts = new Map();
+contexts.set(vcExamplesV1CtxUrl, vcExamplesV1Ctx);
+contexts.set(odrlCtxUrl, odrlCtx);
+realContexts.forEach((value, key) => contexts.set(key, value));
```

Comment:
This bit seems like it's breaking abstractions. It would be better to use the pattern in the README to extend the document loader.

Reply:
Updated test context loader to pattern used in README. [7b2cb4f](https://github.com/digitalbazaar/vc/pull/117/commits/7b2cb4f576f6f8ae63e3622b460c3825a1bd6791)